Makefile

```
@@ -13,6 +13,8 @@ pam-sys: $(PAM_SRC_DIR)/sys.rs
 
 $(PAM_SRC_DIR)/sys.rs: $(PAM_SRC_DIR)/wrapper.h
 	$(BINDGEN_CMD) $< --output $@
+	cargo minify --apply --allow-dirty
+	sed -i 's/rust-bindgen \w*\.\w*\.\w*/\0, minified by cargo-minify/' $@
```

Note: due to differences between FreeBSD and GNU, this sed invocation will only work on Linux.

On FreeBSD the `-i` flag expects an argument that provides the suffix for the "backup" file. 

What would work cross-platform is:
```suggestion
	sed -i.bak 's/rust-bindgen \w*\.\w*\.\w*/\0, minified by cargo-minify/' $@
```
(This will however create a ".bak" file for each modified file; I don't know if that impacts the git command that we use in the CI)

---

It should be possible to remove the .bak file again immediately after the sed command. In any case for FreeBSD some changes to the Makefile will be necessary anyway. FreeBSD's make seems to not support `$<` and it is necessary to make it write to a different file on Linux and FreeBSD.

---

For this simple makefile it should be possible to write a cross-platform one, but if there are other blockers we can resolve that at a later point and leave this as is for the moment, yes.